src/styles/blocks/item/item__blocks/reviews.scss

```diff
@@ -0,0 +1,35 @@
+.card {
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__stars {
+    display: flex;
+    gap: 4px;
+  }
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    width: 16px;
+    height: 16px;
+  }
+
+  &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
You should reuse logic from task stars.
If I add class stars--2 I should see only 2 active stars
The same with other modifiers